tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "compilerOptions": {
+    "lib": ["esnext", "dom"],
+    "types": ["reflect-metadata"],
+    "outDir": "./built",
+    "allowJs": true,
+    "target": "es5",
```

Comment:
Is there a benefit targetting es5 over es6?

Reply:
Huh not that I can think of. I'm just so use to targeting browsers that it was automatic.

Reply:
makes sense. I'll bet that the compiled code will be a lot more readable if it's es6 (but ideally we never look at the compiled code?)